lib/webhookdb/brevo.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Developer Reference: https://developers.brevo.com/reference
+# Webhooks are supported but not in all: https://developers.brevo.com/docs/how-to-use-webhooks
+class Webhookdb::Brevo
+  extend Webhookdb::MethodUtilities
+  include Appydays::Configurable
+  include Appydays::Loggable
+
+  configurable(:brevo) do
+    setting :http_timeout, 30
+    # See https://developers.brevo.com/docs/how-to-use-webhooks#securing-your-webhooks
+    # for the originating IPs.
+    setting :allowed_ip_blocks, %w[185.107.232.1/24 1.179.112.1/20],
```

Comment:
Good job- normally webhooks are secured with cryptography, this is a new one!

Reply:
Please use normal string arrays (ie, `['a', 'b']`), not `%w` (sometimes we use `%r{my regex}` for regex, and `%(my multiline string)` for multiline strings, but that's it. I understand this distinction is arbitrary but better to be consistent, I cannot tell you what `%w[]` was without testing it.